### PR TITLE
fix(filmmodu): update domain

### PR DESCRIPTION
**Pull request by 4Fast**
Version & domain updated

### CODE REVIEW

File: websites/F/filmmodu/dist/metadata.json [hunk 1]
```diff
@@ -4,14 +4,20 @@
 		"name": "EGGSY",
 		"id": "162969778699501569"
 	},
+	"contributors": [
+		{
+			"name": "Fast",
+			"id": "308087335873019905"
+		}
+	],
```
Comment (ImRodry) [inline websites/F/filmmodu/dist/metadata.json:12]: you didn't contribute to the code so you cannot be added to the contributors

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: websites/F/filmmodu/dist/metadata.json (hunk 1 above)
Comment (4Fast) [reply websites/F/filmmodu/dist/metadata.json]: Fixed

**Comment by 4Fast** [comment]
Fixed

**Review by ImRodry — DISMISSED** [review]

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by Slowlife01 — CHANGES_REQUESTED** [review]
Provide proof of it working

**Pull request closed without merging**